tests/test_state.py

```diff
@@ -27,6 +27,12 @@ def test_read_dict_state():
         assert isinstance(state, DictState)
 
 
+def test_read_asreview_file():
+    state_fp = Path("tests", "state_files", "test_1_inst.asreview")
+    with state_from_asreview_file(str(state_fp)) as state:
+        assert isinstance(state, JSONState)
```

Comment:
I think the test syntax should be (there is no context manager involved in `state_from_asreview_file`):

```
    with open_state(state_fp) as state:
        assert isinstance(state, JSONState)
```

Reply:
Hmm, maybe this is oke. Ideally, both situations should work. See the other comment as well. 

For example https://github.com/asreview/asreview/blob/cc7d15e99d00abc511880b9fb78a09d80633c8b2/asreview/analysis/analysis.py#L107 can make use of open_state, isn't it? Closing the state file will be safer.